shared/chat/inbox/row/teams-divider/index.tsx

```diff
@@ -25,16 +25,14 @@ class TeamsDivider extends React.PureComponent<Props> {
         gapEnd={true}
       >
         {this.props.showButton && (
-          <Kb.ClickableBox onClick={this.props.toggle} style={styles.containerToggleButton}>
```

Comment:
Can we remove any unused styles from here?

Reply:
Oh, right. Good call.